packages/perseus-core/src/accessibility.ts

```diff
@@ -8,27 +8,23 @@ import _ from "underscore";
 import {traverse} from "./traversal";
 import * as Widgets from "./widgets/core-widget-registry";
 
-// Iterate over a single Perseus renderer, mutating `widgets` by appending
-// violating widget types discovered in this item.
-function traverseRenderer(itemData, widgets: Array<any>) {
-    traverse(itemData, null, function (info) {
-        if (info.type && !Widgets.isAccessible(info.type, info.options)) {
-            widgets.push(info.type);
-        }
-    });
-}
```

Comment:
I inlined this function because:

* it didn't have a very good/descriptive name
* it had only one usage which actually felt clearer when inlined